client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+
```

Comment:
There are some errors on the console. please try fixing them

Reply:
👍